polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -845,10 +845,27 @@ struct TypeAlignOpLowering : public ConvertOpToLLVMPattern<TypeAlignOp> {
   }
 };
 
+static Optional<Type> convertBodyType(llvm::ArrayRef<mlir::Type> body,
+                                      LLVMTypeConverter &converter) {
+  SmallVector<Type> convertedElemTypes;
+  convertedElemTypes.reserve(body.size());
+  if (failed(converter.convertTypes(body, convertedElemTypes)))
+    return std::nullopt;
+  return LLVM::LLVMStructType::getLiteral(&converter.getContext(), body);
+}
+
+static void
+populatePolygeistToLLVMTypeConversion(LLVMTypeConverter &typeConverter) {
+  typeConverter.addConversion([&](polygeist::StructType type) {
+    return convertBodyType(type.getBody(), typeConverter);
```

Comment:
Given that `convertBodyType` is only used here can you inline it?

Reply:
done.